monte_carlo.py

```diff
@@ -158,12 +159,18 @@ def transport_neutron(materials, bounds, tallies, fission_banks, first_round,
             neutron_distance -= tempd
 
             # add distance to cell flux
-            mesh.flux_add(cell, tempd)
+            mesh.flux_add(neutron.group, cell, tempd)
```

Comment:
Let's be consistent with the ordering of arguments. In the previous change above, group is listed last. Here, group is listed first. I don't care which way the arguments are ordered but let's pick an ordering and be consistent with it.